### PR TITLE
Create ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by daj

Ticket: [MTM-59456](https://cumulocity.atlassian.net/browse/MTM-59456)
Original PR: [6881](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6881)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix permissions check in canEditDashboard (#6881)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Fix permissions check when editing dashboards
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59456
+version: 1018.503.116
+---
+fix permissions check in canEditDashboard (#6881)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, users were unable to edit dashboards even if they had the appropriate permissions. This was due to an incorrect permissions check in the canEditDashboard function. The permissions check has now been fixed, allowing users with the correct permissions to edit dashboards as expected. This change ensures that the dashboard editing functionality aligns with the user's assigned permissions, providing a more consistent and secure experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59456
+version: 1018.503.116
+---
+Fixed dashboard delete button permissions check to use correct roles. Previously, the button was enabled for users without deletion rights, causing confusion. Now, it's only enabled for users with proper permissions, accurately reflecting access rights.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md:17]:
```suggestion
Previously, users could delete dashboards without having the required permissions. This issue has been fixed. Now the **Delete** button is only enabled for users with the appropriate permissions, accurately reflecting the access rights.
```

**Review by BeateRixen — APPROVED** [review]

**Review by DawidSAG — APPROVED** [review]

**Pull request merged**